TestProjects/HDRP_RuntimeTests/Assets/HDRPDefaultResources/HDRenderPipelineGlobalSettings.asset

```diff
@@ -62,8 +62,7 @@ MonoBehaviour:
     materialQuality: 0
   m_RenderPipelineResources: {fileID: 11400000, guid: 3ce144cff5783da45aa5d4fdc2da14b7,
     type: 2}
-  m_RenderPipelineRayTracingResources: {fileID: 11400000, guid: ba6e7f30e5cffc249a8bf7ee5756c196,
-    type: 2}
+  m_RenderPipelineRayTracingResources: {fileID: 0}
```

Comment:
This happened automatically because `ClearRayTracingResources()` was called - @jenniferd-unity is this expected?

Reply:
We could revert this portion yes

Reply:
you shouldn't commit this file anyway

Reply:
This file must be committed. HDRP & URP runtime tests assert unless global settings asset is up-to-date. However, I can revert the ray tracing resources part if desired.